packages/azure-kusto-ingest/src/managedStreamingIngestClient.ts

```diff
@@ -175,7 +174,7 @@ class KustoManagedStreamingIngestClient extends AbstractKustoClient {
                 }
             }
 
-            stream = isBlob ? undefined : isNode ? streamify([stream]) : (descriptor as StreamDescriptor).stream;
+            stream = isBlob ? undefined : isNode ? readableToStream(stream!) : (descriptor as StreamDescriptor).stream;
```

Comment:
- Add () for clarification
- Can you guarantee that stream is not null when it does !?

Reply:
add () where

Reply:
yes i can guarantee